deploy/shakenfist_ci/tests/test_upgrades.py

```diff
@@ -4,170 +4,18 @@
 class TestUpgrades(base.BaseTestCase):
     def test_upgraded_data_exists(self):
         # There is an upgraded namespace called 'nstest'
-        self.assertIn('nstest', self.system_client.get_namespaces())
+        self.assertIn('upgrade', self.system_client.get_namespaces())
 
         # Collect networks
         networks = {}
         for net in self.system_client.get_networks():
-            networks[net['uuid']] = net
+            networks['%s/%s' % (net['namespace'], net['name'])] = net
 
-        self.assertIn('30acadd4-66ee-4390-9477-e31d11fe44cb', networks)
-        self.assertIn('afce2cce-dec2-4cf7-a61b-2c4614b320c7', networks)
-
-        # This network is in the dump, but removed at upgrade for having
-        # and invalid netblock.
-        self.assertNotIn('3edb03de-4f3d-4859-b822-94924073a1ea', networks)
-
-        # First network
-        self.assertEqual(
-            'testnet', networks['30acadd4-66ee-4390-9477-e31d11fe44cb']['name'])
-        self.assertEqual(
-            'system', networks['30acadd4-66ee-4390-9477-e31d11fe44cb']['namespace'])
-        self.assertEqual(
-            '192.16.10.0/24', networks['30acadd4-66ee-4390-9477-e31d11fe44cb']['netblock'])
-
-        # Second network
-        self.assertEqual(
-            'nsnet', networks['afce2cce-dec2-4cf7-a61b-2c4614b320c7']['name'])
-        self.assertEqual(
-            'nstest', networks['afce2cce-dec2-4cf7-a61b-2c4614b320c7']['namespace'])
-        self.assertEqual(
-            '192.168.20.0/24', networks['afce2cce-dec2-4cf7-a61b-2c4614b320c7']['netblock'])
+        self.assertIn('upgrade/upgrade', networks)
```

Comment:
Would be good to test the state of the network.

Reply:
Sorry, done but I didn't reply.